bot/__main__.py

```diff
@@ -84,12 +87,7 @@ def printWithTime(msg):
 		if val in message.content:
 			sleep_time = 6  # tijd dat user gemute is
 			# Find user or mute jack if none found
-			userID = findID(message.content)
-			if userID is None or userID == jack:
-				userID = jack
-				sleep_time = 10
-				printWithTime("No user found. Selecting jack...")
-		
+			userID = findID(message.content)		
```

Comment:
geeft het nu geen error als het geen gebruiker vind?

Reply:
nvm, dat heb je opgelost in findID

Reply:
bij geen user returnt hij standaard jack. zou geen probleem moeten zijn